src/acl_mem.cpp

```diff
@@ -965,6 +965,16 @@ CL_API_ENTRY cl_mem CL_API_CALL clCreateBufferIntelFPGA(cl_context context,
                                            errcode_ret);
 }
 
+// Create a buffer with property
+// It is equivalent to clCreateBufferWithPropertiesINTEL
```

Comment:
Nitpick: This comment duplicates the content of the function body, so you could remove it if you like. It really does not matter nor apply here, but in general, not duplicating the more obvious facts in comments avoids comment and code drifting apart eventually. Omitting the obvious facts may also motivate to ponder a bit deeper about the value a comment could provide to the reader (which again does not apply here).